setup.cfg

```diff
@@ -39,10 +39,11 @@ install_requires =
     pluggy>=0.12.0
     py>=1.4.17
     six>=1.14.0 # required when virtualenv>=20
-    toml>=0.9.4
     virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0
     colorama>=0.4.1 ;platform_system=="Windows"
     importlib-metadata>=0.12;python_version<"3.8"
+    toml>=0.9.4;python_version<="3.5"
```

Comment:
0.10.2 here per https://pypi.org/project/toml/

Reply:
IIRC version 2 does not support Python 3.6 so that requires using toml on 3.6. Is that what you prefer?

Reply:
I don't understand this request. It was already pinned to a different version and I kept it the same. I can change it, but I wonder why.

Reply:
Done.